elements/rh-pagination/docs/20-guidelines.md

```diff
@@ -68,116 +45,166 @@ In some edge cases, pagination may need to display a very low page count like on
+      <img src="{{ '../pagination-guidelines-best-practices-1-do.svg' | url }}" alt="Size SM truncated pagination with input below pages.">
+    </uxdot-example>
+    <p>The page input field needs to be visible when there is truncation or when only the navigation controls are available.</p>
+  </uxdot-best-practice>
+  <uxdot-best-practice dont>
+    <uxdot-example slot="image" width-adjustment="372px">
+      <img src="../pagination-guidelines-best-practices-1-dont.svg" alt="Size SM truncated pagination without an input.">
+    </uxdot-example>
+    <p>Do not omit the page input field when users do not have the ability to navigate to every page quickly.</p>
+  </uxdot-best-practice>
+</div>
 
 ### Truncation
 
-Do not truncate pagination if there is less than seven pages.
-
-<uxdot-example width-adjustment="414px" danger>
-  <img src="{{ '../pagination-best-practice-3.png' | url }}" alt="Image of pagination that is truncating only four pages is incorrect usage">
-</uxdot-example>
+<div class="grid sm-two-columns">
+  <uxdot-best-practice do>
+    <uxdot-example slot="image" width-adjustment="372px">
+      <img src="{{ '../pagination-guidelines-best-practices-2-do.svg' | url }}" alt="Size SM truncated pagination with eight pages.">
+    </uxdot-example>
+    <p>Truncation occurs automatically when there are more than seven pages.</p>
+  </uxdot-best-practice>
+  <uxdot-best-practice dont>
+    <uxdot-example slot="image" width-adjustment="270px">
+      <img src="../pagination-guidelines-best-practices-2-dont.svg" alt="Pagination that is truncating only four pages is incorrect usage">
+    </uxdot-example>
+    <p>Do not truncate pagination if there are fewer than seven pages.</p>
+  </uxdot-best-practice>
+</div>
 
 
 ### Order or alignment
 
-Do not change the order or alignment of the page input field.
-
-<uxdot-example width-adjustment="794px" danger>
-  <img src="{{ '../pagination-best-practice-4.png' | url }}" alt="Image of paginations with page input fields; one group shows incorrect order and the other group shows incorrect alignment">
-</uxdot-example>
+<div class="grid sm-two-columns">
+  <uxdot-best-practice do>
+    <uxdot-example slot="image" width-adjustment="372px">
+      <img src="{{ '../pagination-guidelines-best-practices-3-do.svg' | url }}" alt="Pagination with input field center aligned below pagination">
+    </uxdot-example>
+    <p>Use the built-in alignment options for the page input field. If your language is read right to left, view the “Right to left” pagination demo.</p>
+  </uxdot-best-practice>
+  <uxdot-best-practice dont>
+    <uxdot-example slot="image" width-adjustment="372px">
+      <img src="../pagination-guidelines-best-practices-3-dont.svg" alt="Truncated pagination with the input field on top and right aligned.">
+    </uxdot-example>
+    <p>Do not change the order or alignment of the page input field.</p>
+  </uxdot-best-practice>
+</div>
+
+
+### Compact vs. mobile
```

Comment:
Is it possible to remove the top padding for these two Do/Don't images?

Reply:
Resolved in [b6308e3](https://github.com/RedHat-UX/red-hat-design-system/pull/1587/commits/b6308e3acd5d4e3737d2a0a940decf28f39ca1a5).